icons/elixir.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Elixir icon</title><path d="M19.789,16.578c0,3.75-2.926,7.422-7.739,7.422c-5.246,0-7.839-3.708-7.839-8.285 c0-5.207,3.89-12.946,7.995-15.639c0.277-0.182,0.643,0.041,0.611,0.371c-0.03,0.313-0.046,0.631-0.046,0.951 c0,2.067,0.641,3.985,1.738,5.563c0.522,0.795,1.092,1.477,1.763,2.351c0.94,1.226,1.636,1.905,2.642,3.84 c0.005,0.01,0.01,0.018,0.015,0.028C19.48,14.197,19.789,15.352,19.789,16.578z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Elixir icon</title><path d="M19.7931 16.5744c0 3.7513-2.9272 7.4256-7.7426 7.4256-5.2484 0-7.8436-3.7103-7.8436-8.2897 0-5.2104 3.8923-12.953 8.0006-15.6472a.397.397 0 01.61.371c-.03.314-.046.632-.046.9521 0 2.0682.6421 3.9873 1.7402 5.5654.521.7961 1.092 1.4782 1.7631 2.3522.9401 1.2271 1.6372 1.9062 2.6432 3.8423l.015.028a7.1076 7.1076 0 01.8601 3.4003z"/></svg>
```

Comment:
Looks spot on! Will this work with a precision of 3?

Reply:
It worked 😉